core/primitives/src/version.rs

```diff
@@ -141,6 +143,9 @@ impl ProtocolFeature {
             ProtocolFeature::BlockHeaderV3 => 109,
             #[cfg(feature = "protocol_feature_restore_receipts_after_fix")]
             ProtocolFeature::RestoreReceiptsAfterFix => 112,
+            #[cfg(feature = "protocol_feature_simple_nightshade")]
+            // do not enable it yet
+            ProtocolFeature::SimpleNightshade => 200,
```

Comment:
Ah initially it's because I don't want to enable this feature yet. This PR only implements part of the change for splitting states for simple nightshade. I set it to a super high number to prevent from accidentally enabling it. But now I realized I do want to enable it for nightly testing as I implemented some unit test for this feature already. I will change to a lower number